core/templates/pages/learner-group-pages/create-group/add-syllabus-items.component.html

```diff
@@ -0,0 +1,206 @@
+<div class="oppia-add-learner-group-syllabus">
+  <div class="add-learner-group-syllabus-desc-text">
+    Add skills or stories to your syllabus to automatically send them to your students.
```

Comment:
Add this to translation. Ditto elsewhere

Reply:
Done